aten/src/ATen/Dispatch.h

```diff
@@ -159,6 +159,7 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
       AT_PRIVATE_CASE_TYPE(at::ScalarType::Int, int32_t, __VA_ARGS__)        \
       AT_PRIVATE_CASE_TYPE(at::ScalarType::Long, int64_t, __VA_ARGS__)       \
       AT_PRIVATE_CASE_TYPE(at::ScalarType::Short, int16_t, __VA_ARGS__)      \
+      AT_PRIVATE_CASE_TYPE(at::ScalarType::QInt8, uint8_t, __VA_ARGS__)      \
```

Comment:
don't do this -- I know it's called ALL_TYPES, but it means something like ALL_STANDARD_TYPES.  The issue is that extensions use this and you will break them if you add another type.